pyproject.toml

```diff
@@ -32,4 +32,10 @@ ignore = [
 
 [tool.ruff.lint.isort]
 force-single-line = false
-known-first-party = ["openfl"]
\ No newline at end of file
+known-first-party = ["openfl"]
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git*,*.svg,*.css'
+check-hidden = true
+ignore-regex = '^\s*"image/\S+": ".*'
+ignore-words-list = 'fpr,astroid'
```

Comment:
please remove

Reply:
will do, although this is "your loss" happen you decide to run the tool to catch fresh typos